### PR TITLE
add grpcio bench

### LINKED ISSUE #107: Add new rust impl in bench
Hi, guys.
You can add new bench case rust with https://github.com/tikv/grpc-rs ?

### PR DESCRIPTION
**Pull request by Trisfald**
closes #107

### CODE REVIEW

File: rust_grpcio_bench/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+use crate::proto::gen::helloworld_grpc::{create_greeter, Greeter};
+
+#[derive(Clone)]
+struct GreeterService;
+
+impl Greeter for GreeterService {
+    fn say_hello(&mut self, ctx: RpcContext<'_>, req: HelloRequest, sink: UnarySink<HelloReply>) {
+        let msg = format!("{}", req.get_name());
+        let mut resp = HelloReply::default();
+        resp.set_message(msg);
```
Comment (LesnyRumcajs) [inline rust_grpcio_bench/src/main.rs:22]: wouldn't it be faster to just pass `req.get_name().clone()` directly?

Comment (Trisfald) [reply rust_grpcio_bench/src/main.rs:22]: maybe
but it's cleaner anyway

Comment (LesnyRumcajs) [reply rust_grpcio_bench/src/main.rs:22]: How about `let msg = req.get_name().clone()`? My point is, I'm not sure this format macro is going to nicely optimize itself to just clone the underlying string. Or will it?

**Review by LesnyRumcajs — APPROVED** [review]

File: rust_grpcio_bench/src/main.rs (hunk 1 above)
Comment (Trisfald) [reply rust_grpcio_bench/src/main.rs]: Yeah that's what I did 🚀

**Pull request merged**